css/style.css

```diff
@@ -1,3 +1,121 @@
-h1 {
+/*h1 {
   color: salmon;
+}*/
+
+body {
+  background-color: #e9ebe7;
+  color: #43474b;
+  /*font-family: "Hind", sans-serif;*/
+  font-family: "Helvetica", sans-serif;
+  font-size: 0.7em;
+}
+
+h1 {
+  margin-top: 40px;
+  /*font-weight: normal;*/
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /*justify-content: space-around;*/
+}
+
+.Guidebooks, .Weekend {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 30px;
+  color: white;
+  font-size: 1.5em;
+  text-align: center;
+}
+ .Meet {
+   text-align: center;
+   margin-bottom: 10px;
+ }
+.Just {
+  margin-bottom: 10px;
+  text-align: center;
+
+}
+.sanfrancisco{
```

Comment:
Since your city image divs share a lot of properties in common, it makes sense to separate them out into one css selector to avoid duplication. Something like this:

```css
.city {
   width: 230px;
   height: 250px;
   margin: 10px;
   background-size: cover;
   font-weight: bold;
   font-size: 20px;
}

.sanfrancisco{
   background: url(../img/san-francisco.jpg);
}

.newyork {
   background: url(../img/new-york.jpg);
}
...
```